code/WorkInProgress/Mini/ATM.dm

```diff
@@ -550,7 +554,21 @@ log transactions
 			dispense_cash(arbitrary_sum,H.wear_id)
 			to_chat(usr, "[bicon(src)]<span class='notice'>Funds were transferred into your physical wallet!</span>")
 			return
-	dispense_cash(arbitrary_sum,get_step(get_turf(src),turn(dir,180))) // Spawn on the ATM.
+		if(istype(H.wear_id, /obj/item/device/pda))
+			var/obj/item/device/pda/P = H.wear_id
+			to_chat(usr, "[bicon(src)]<span class='notice'>Funds were transferred into your virtual wallet!</span>")
+			P.add_to_virtual_wallet(arbitrary_sum, user)
+			return
+	var/turf/our_turf = get_turf(src)
+	var/turf/destination_turf = get_step(our_turf, turn(dir, 180))
```

Comment:
would it not be more reliable to just throw it toward user?

Reply:
Stepping towards user may put it in the "consume" turf